app/models/manageiq/providers/cloud_manager/virtual_template.rb

```diff
@@ -0,0 +1,14 @@
+class ManageIQ::Providers::CloudManager::VirtualTemplate < ::MiqTemplate
+  validate :single_template, :on => :create
+  default_value_for :cloud, true
+
+  TYPES = {
+    :amazon => 'ManageIQ::Providers::Amazon::CloudManager::VirtualTemplate'
+  }.freeze
+
+  def single_template
+    multiple = !type.constantize.where(:type => type).empty?
+    errors.add(:virtual_template, 'may only have one per type') if multiple
```

Comment:
@chriskacerguis I'm not sure api error messages are placed into translations from what I can tell